components/apimgt/org.wso2.carbon.apimgt.cleanup.service/src/main/java/org/wso2/carbon/apimgt/cleanup/service/IdpKeyMangerPurge.java

```diff
@@ -105,7 +112,8 @@ private void initTaskList() {
         }
 
         APIUtil.logAuditMessage(APIConstants.AuditLogConstants.ORGANIZATION,
-                new Gson().toJson(IdpKeyMangerPurgeTaskMap), APIConstants.AuditLogConstants.DELETED, username);
+                new Gson().toJson(IdpKeyMangerPurgeTaskMap), APIConstants.AuditLogConstants.DELETED,
+                "Organization-Cleanup-Executor");
```

Comment:
This is used in multiple places, Why not use a constant?

Reply:
Added to constant file